package.json

```diff
@@ -100,6 +100,7 @@
     "mocha": "^10.2.0",
     "playwright": "^1.30.0",
     "rollup": "^3.29.4",
+    "rollup-plugin-typescript2": "^0.36.0",
```

Comment:
we went with this one because it prints typeScript diagnostic messages when bundling the code while the official ones does not and I feel this why we wanted to move to typescript in the first place